napari/layers/labels/labels.py

```diff
@@ -113,6 +113,8 @@ class Labels(_ImageBase):
         'translucent' renders without lighting. 'iso_categorical' uses isosurface
         rendering to calculate lighting effects on labeled surfaces.
         The default value is 'iso_categorical'.
+    depiction : str
+        3D Depiction mode used by vispy. Must be one of our supported modes.
```

Comment:
I don't think we should refer to vispy here as I think it will be confusing to most users (they might not be aware that vispy is doing the rendering).

```suggestion
        3D Depiction mode. Must be one of our supported modes.
```